### DISCUSSION TITLE
New payment system replacing sync on blockchain

### DISCUSSION THREAD

**Discussion by varrt** [sunflower-land/sunflower-land#2204] [Ideas]
**What is a problem?**

Right now, as users we need to sync on chain to restore supply for seeds and tools. Also we have to do this to see our SFL balance, resources and NFT on Goblin Retreat. To sync on chain we have to pay a fee for the Team and also network gas. Right now, the Team get 0.10$ (0.08 MATIC) and the rest is network gas. The problem is network gas that sometimes is horribly high. Often it is more than fee for the Team. In the final effect we paid more for miners than Sunflower Land creators.

I understand the fee for miners depends on how much data we send in a transaction. That makes a lot of fees for high level farms, where they have to do a lot of things between syncs.

**Solution**

I propose a new system where players don’t have to do syncs so often. 

1. Give up with sync on chain to restore supply of seeds and tools or extremely increment supplies.
2. Implement monthly subscriptions or season pass for players. For example 5$ in MATIC. It’s the same value for the team like 50 syncs. To proof the player bought a season pass will be some kind of ticket as NFT. For new players they give this pass for a month when they mint a land.
3. Left players choose to sync when they want to trade with other players or mint NFT for resources.
4. To force players to make sync once per week they must sync to reset grub shop (but only once per week).

**Potential problems**

I am not sure it’s ok to not sync with blockchain for all week. I don’t know if it created new problems on the technical side.
How to force players to make syncs more often. Not only by resetting the grub shop.

**Comment by IvanSkazkin** [comment]
Honestly, I came to write something probable.
For example, you can do. So. For 1 synchronization of payment 5-10 updates in the store.
So 10 syncs will cost $2 (or more). But synchronization in the context will be once.
